content/ember-data-polaris-2023-04-11.md

```diff
@@ -0,0 +1,74 @@
+---
+title: EmberData Polaris Update
+authors:
+  - chris-thoburn
+  - the-ember-data-team
+date: 2023-04-11T00:00:00.000Z
+tags:
+  - '2023'
+  - ember-data
+  - ember-polaris
+
+---
+
+This week, in coordination with the broader Ember project, EmberData released 4.12, its final
+4.x version, and began iterating towards the first release of the 5.x series.
+
+## A wrap on 4.x
+
+This final release of the 4.x cycle introduced several new concepts to support a new paradigm for managing [Requests](https://github.com/emberjs/data/tree/v4.12.0/packages/request#readme) and new capabilities for [Caching Data](https://api.emberjs.com/ember-data/release/classes/%3CInterface%3E%20Cache).
+
+The primary focus of these changes was to support caching **Documents** and to ensure that **Resources** are handled opaquely. Loosely defined: A Document encompasses the full response returned by an API request, and can typically be identified uniquely by url. Documents may contain many resources or references to resources, where a resource is defined as some data which your application or the API treats as uniquely identitifiable.
+
+> *Note:* Historically, Model has mapped to a resource "type" and serves to present the data for a resource. It may be useful to think about resources as the backing data for a Model; however, resources can be far more generic than this and could even be anything. This includes raw strings, html, xml, buffers, images, blobs, and streams!
```

Comment:
I like the way this looks as a block because of the styling, but the block styling also sorta makes it look like we're quoting from somewhere else because of the quote icon it employs. Not really a blocker, just something to consider.

Reply:
yeah I thought the same. It's definitely an aside, not sure if we have a good way to easily style that ...

Reply:
Was looking for the styling of `EsNote` from the ember-styleguide, but it seems broken here in the blog. I'll have to look into that.